documentation/README.md

```diff
@@ -0,0 +1,83 @@
+# TL;DR
+
+The main documentation of the Findex server is in [docs/index.md](./docs/index.md)
+
+## Installing Rendering Tools
+
+You **do not need these tools** to author the doc and should probably **not** install LaTeX.
+Vs-code has all the necessary support for it.
+
+However, if you want to see how it looks fully rendered, you must install `mkdocs`
+
+### Installing mkdocs
```

Comment:
Why do we rely on mkdocs ? I really do not appreciate these extra dependencies. And I hate venvs.

Reply:
There are 2 types of docs:
- the github doc (only markdown)
- the Cosmian public doc (mkdocs). 
`mkdocs` is inherited from the public_doc and this current doc is merged in public_doc automatically